### DISCUSSION TITLE
More convenient links to achievement hunting rules

### DISCUSSION THREAD

**Discussion by Nepiki** [RetroAchievements/RAWeb#2821] [Ideas]
I should preface this discussion by saying that this is mostly to help visibility. When creating an account, users agree to the general terms which includes the rules, and they have no excuse to not know about them whenever action is taken. But I do think there are some ways we can improve visibility to them, as the user currently needs to jump through a few hoops to get there: either privacy policy => users code of conduct => global achievement hunting rules, or scroll down to the bottom of the page and there go from users code of conduct => global achievement hunting rules.

**Suggestion 1**
An additional checkbox on account creation that must be ticked, with a direct link to the achievement hunting rules. While most users still tend to just click the checkbox and move on, it would cover our bases just a bit more.

![Example1](https://github.com/user-attachments/assets/726f91ee-9641-467f-a88f-7f550b4b36ba)

**Suggestion 2**
Similar to suggestion 1, but there is a pop-up after the user clicks ''create account''. May not be desirable from an UX standpoint however.

![Suggestion2](https://github.com/user-attachments/assets/faf10075-8eea-4bcb-91f1-8b7b31a7ab11)

**Suggestion 3**
The most simple of solutions, an additional direct link on the homepage to the achievement hunting rules.

![Example 3](https://github.com/user-attachments/assets/cf05016e-6e0c-4bf8-b03e-c468c9c06518)

**Comment by iTom33ro** [comment]
I agree with this! Another suggestion that I have is to put the additional link to the topbar of the website. That way, it would be visible in every page.

**Comment by wescopeland** [comment]
I agree we should have more visibility to these rules. I've considered each of these items proposed already and have some counterpoints as well as alternative ideas.

> Suggestion 1
An additional checkbox on account creation that must be ticked, with a direct link to the achievement hunting rules. While most users still tend to just click the checkbox and move on, it would cover our bases just a bit more.

This would only be visible to new users, and our data supports that this would likely go unread. According to our analytics, only 0.0004% of all monthly unique traffic navigates to our terms page.

---

> Suggestion 2
Similar to suggestion 1, but there is a pop-up after the user clicks ''create account''. May not be desirable from an UX standpoint however.

I'm not a fan of this for two reasons:

1. If and when the rules change, they now need to be changed in two places and we need to kick off a deploy to update this modal content.
2. It's an open secret that most of these rules don't apply to softcore users.

---

> Suggestion 3
The most simple of solutions, an additional direct link on the homepage to the achievement hunting rules.

This could be viable. My main concern is the more stuff we put over here, the lower CTR we'll likely see for everything else there, as any new item slightly deemphasizes all the others. This link is also currently buried for mobile users.

I can't actually prove this yet, but the home page rewrite PR does add tracking to all these buttons so we can begin running experiments and draw some conclusions from the data.

---

A fourth alternative I might propose is if a user crosses a certain score or rank threshold, we then pop some kind of modal when they visit the site (almost like a notification) showing them a link to the rules. I like this idea because:
* It's specific to the user's context. Casual players will likely never see this point of friction.
* It could be a non-persistent element. Once the user sees it and confirms it, they don't see it again.
* It would be visible to both desktop and mobile users.

Another option would be to include a link to the rules somewhere on all leaderboard pages, ie: the global points leaderboard and the beaten games leaderboard.
